docs/index.rst

```diff
@@ -15,35 +15,35 @@ The core features include:
    :maxdepth: 1
    :caption: Getting Started
 
-   install.md
-   send_request.ipynb
+   starts/install.md
+   starts/send_request.ipynb
 
 
 .. toctree::
    :maxdepth: 1
    :caption: Backend Tutorial
 
-   openai_api.ipynb
-   backend.md
+   backend/openai_api.ipynb
```

Comment:
do we also need to update these links? https://github.com/sgl-project/sglang/blob/b9fd178f1b7bab721b384c017dcec30a3ba0f323/README.md?plain=1#L42-L49

Reply:
Let me check this.

Reply:
Yeah. We need to do this @merrymercy